docs/content/integrations/airflow/migrating-to-dagster.mdx

```diff
@@ -13,99 +13,208 @@ description: guide for doing a lift and shift migration of airflow to dagster
+
+- **To know the following about your Airflow setup**:
+
+  - What operator types are used in the DAGs you're migrating
+  - What Airflow connections your DAGs depend on
+  - What Airflow variables you've set
+  - What Airflow secrets backend you use
+  - Where the permissions that your DAGs depend on are defined
+
+  If you're unsure where to find these things, you're likely not the right person to make the migration. We recommend looping a teammate in before continuing.
+
+- **If using Dagster Cloud**, an existing [Dagster Cloud](/dagster-cloud) account. While your migrated Airflow DAGs will work with Dagster Open Source, this guide includes setup specific to Dagster Cloud.
+
+  **If you just signed up for a Cloud account**, follow the steps in the [Dagster Cloud Getting Started guide](/dagster-cloud/getting-started) before proceeding.
+
+---
+
+### Before you begin
+
+You may be coming to this document/library in a skeptical frame of mind, having previously been burned by projects that claim to have 100% foolproof, automated migration tools. We want to assure you that we are _not_ making that claim.
+
+The `dagster-airflow` migration library should be viewed as a powerful _accelerant_ of migration, rather than guaranteeing completely _automated_ migration. The amount of work required is proportional to the complexity of your installation. Smaller implementations of less complexity can be trivial to migrate, making it appear virtually automatic; larger, more complicated, or customized implementations require more investment.
+
+For larger installations, teams that already adopt devops practices and/or have standardized usage of Airflow will have a smoother transition.
+
+Some concrete examples:
+
+- If you rely on the usage of the Airflow UI to set production connections or variables, this will require a change of workflow as you will no longer have the Airflow UI at the end of this migration. If instead you rely on code-as-infrastructure patterns to set connections or variables, migration is more straightforward.
+- If you have standardized on a small set or even a single operator type (e.g. the `K8sPodOperator`), migration will be easier. If you use a large number of operator types with a wide range of infrastructure requirements, migration will be more work.
+- If you dynamically generate your Airflow DAGs from a higher-level API or DSL (e.g. yaml), the migration will be more straightforward than all your stakeholders directly creating Airflow DAGs.
+
+Even in the case that requires some infrastructure investment, `dagster-airflow` dramatically eases migration, typically by orders of magnitude. The cost can be borne by a single or small set of infrastructure-oriented engineers, as it dramatically reduces coordination costs: You do not have to move all of your stakeholders over to new APIs simultaneously. In our experience, practitioners welcome the change, because of the immediate improvement in tooling, stability, and development speed.
```

Comment:
"The cost can be borne by a single or small set of infrastructure-oriented engineers, which dramatically reduces coordination costs"